### DISCUSSION TITLE
Writing Roms to Third-Party Hardware

### DISCUSSION THREAD

**Discussion by Seafarerr** [sanni/cartreader#209] [Need help]
I am interested in making reproduction cartridges for hacks (e.g. SM64 Splitscreen, ALttP Randomizer) and am wondering if this device would be compatible with empty third-party boards. What limitations does this device have in regards to writing roms to cartridges? If it is incapable of writing to third-party boards, would it be possible to modify my device to allow such a feature?

Some examples of third-party boards I knoew of are:
https://retrostage.net/?product=n64-blaster-2-0
https://www.mortoffgames.com/index.php?route=product/product&path=4_294&product_id=901
https://www.muramasaentertainment.com/product/snes-basic-dip42-4mb/

**Comment by Androxilogin** [comment]
I've had luck with purchasing repros off of eBay and rewriting the carts. It is a lot cheaper than buying the board, the rom/ram chips, CIC chips, the cases, screws, etc. in most cases. But you would have to understand the boards, such as Hi/Lo rom, chip sizes and what is compatible as far as repros go.

**Reply by Seafarerr** [reply]
I'm a bit concerned about eBay carts damaging my console since I don't know how they were manufactured, and I'd also have to deal with the old label and being stuck with the grey case.

So there are no limitations on what types of carts can be written to? I was just uncertain because "write" was often omitted in the Supported Systems section of the readme.

**Reply by Dakkaron** [reply]
I only own a Gameboy, so I can only speak for that system, but I guess it would be similar.

Most of the 3rd party boards are actually exactly the same as the chinese repro carts' boards. Either they are based on the same open source design or sometimes the 3rd party boards are ripped from repros.

It is also hardly possible for a cartridge to actually damage the console (but it is totally possible for the cartridge to be damaged during use, because most of them use 3.3V chips, even on 5V devices).

Reflashing them is also not always that easy. As long as the cartridge is wired up in a way that it is flashable through the regular cartridge connector, you will be able to flash it with this cartreader... But you might have to adjust the flash commands. That's not too hard,but it's not easy either. You will need to read the datasheet for your flash chip, find the codes and replace them in the cartreader's source code.

**Comment by Androxilogin** [comment]
I really couldn't say, I've never owned any of those devices. I just know
the eBay route is a bit cheaper.
